src/components/Item/Image.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Thumbnail = styled.img`
+  width: 100%;
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 1;
+  grid-row-end: 7;
+`;
+
+function Image({ src, title }) {
```

Comment:
Again you don't have to do this.
It's perfectly fine to pass src and alt directly in Item.